contrib/nydusify/cmd/nydusify.go

```diff
@@ -700,8 +707,45 @@ func main() {
 				if err != nil {
 					return err
 				} else if backendConfig == "" {
-					// TODO get auth from docker configuration file
-					return errors.Errorf("backend configuration is empty, please specify option '--backend-config'")
+
+					backendType = "registry"
+					parsed, err := reference.ParseNormalizedNamed(c.String("target"))
+
+					if err != nil {
+						return err
+					}
+
+					host := reference.Domain(parsed)
+					repo := reference.Path(parsed)
+
+					config := dockerconfig.LoadDefaultConfigFile(os.Stderr)
```

Comment:
I think we can create a func named `NewRegistryBackendConfig(...) (string, error)` to reuse the part of codes at least, right?

Reply:
Yes, i will try it.